### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by rileywagner**

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -264,10 +271,10 @@ <h6 class="p-3">
                                     </h6>
                                 </div> 
                                 <div class="col-md-3 col-xs-6 team-max-width">
-                                    <img class="rounded-circle" src="images/arizona/headshot-bunting.jpg">
+                                    <img class="rounded-circle" src="images/vanderbilt/headshot-bunting.jpg">
                                     <h6 class="p-3">
                                         Matthew Bunting <br>
-                                        <small class="text-muted">Postdoctoral Researcher, University of Arizona</small>
+                                        <small class="text-muted">Postdoctoral Researcher, Vanderbilt University</small>
```
Comment (liljonnystyle) [inline about.html]: is he a postdoc or a researcher now?

File: about.html [hunk 2]
```diff
@@ -630,6 +637,7 @@ <h6 class="p-3">
                                         <small class="text-muted">Research Group Manager, GM Research</small>
                                     </h6>
                                 </div> 
+<!--                                 TODO update title and affiliation for Maria Laura Delle Monache -->
```
Comment (liljonnystyle) [inline about.html]: ah yes, we should probably put her as a PI at UC Berkeley.

also we need to add her new PhD student, Han Wang (I'll email and CC you)

**Review by liljonnystyle — CHANGES_REQUESTED** [review]
I think a couple new people to add as well.
- Han Wang, PhD student, UCB
- Adit Shah, Undergrad student, UCB
- Mostafa Ameli, Visiting Professor (?), UCB
- Mikail Mahmood, Undergrad student, Rutgers
- Shengquan Xiang, ?, ?
- Lee Smith, TDOT (and remove/move Brad Freeze?)

Maria Laura will also be adding another postdoc in ~1-2 months

File: about.html (hunk 1 above)
Comment (rileywagner) [reply about.html]: updated title to Research & Development Scientist

File: about.html (hunk 2 above)
Comment (rileywagner) [reply about.html]: relocated Maria Laura as a Co-PI at UC Berkeley
Added Han Wang

**Pull request closed without merging**